ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/GPG_signature_verification/tasks/main.yml

```diff
@@ -14,7 +14,7 @@
   win_shell: C:/cygwin64/bin/bash.exe -c "chmod og-rwx ~/.gnupg"
 
 - name: Import GPG Key
-  win_command: gpg --keyserver keyserver.ubuntu.com --recv-keys "{{ GPG_key }}"
+  win_shell: C:/cygwin64/bin/bash.exe -c "gpg --keyserver keyserver.ubuntu.com --recv-keys "{{ GPG_key }}""
```

Comment:
General question, but what's the advantage/need of win_shell over win_command here?

Reply:
I was just aligning it with the commands above it, the commands running in the cygwin shell, appear to prefer the use of win\_shell, I suspect it actually doesn't matter.. It is important that the gpg signing does run in the cygwin shell though, as otherwise GPG isn't available in Windows, and the playbooks and Github Actions both hang during GPG signature verification of Ant.

Reply:
I need to make a similar fix to the binary verification, so i'll try them both using win\_command.

Reply:
Win_Command hangs when running the cygwin bash shell, reverting to win_shell

Reply:
PR for GPG validation is now fixed